src/sentry/conf/server.py

```diff
@@ -446,7 +447,12 @@ def env(key, default='', type=None):
     Queue('digests.scheduling', routing_key='digests.scheduling'),
     Queue('email', routing_key='email'),
     Queue('events.preprocess_event', routing_key='events.preprocess_event'),
+    Queue('events.preprocess_event_from_reprocessing',
```

Comment:
I'd be inclined to rename this queue and the ones below to:
```
events.reprocess.preprocess_event
events.reprocess.process_event
events.reprocess.schedule
```
Seems a little more fitting of description for me.